### PR TITLE
Add UI to open action package metadata file

### PR DESCRIPTION
**Pull request by mmokko**
In case the metadata.json file is not yet created I opted to create it automatically to make it easy for the user.
Other option would be to show an error and guide user to create the file first. Or add information to robotEntry type to check if metadata file is there and then we can show it based on that. But this would make the robot listing slower if a filesystem action needs to be always done to check if file is there when listing the available actions.

I also added the fix for build and publish command to find the correct input dir if there are multiple action packages in the workspace.

### CODE REVIEW

**Review by custompointofview — COMMENTED** [review]
Just some comments about naming and an additional prompt for the user.

File: sema4ai/vscode-client/src/robo/actionPackage.ts [hunk 1]
```diff
@@ -750,3 +767,25 @@ export const createMetadata = async (actionPackagePath?: vscode.Uri) => {
...
+            return;
+        }
+    }
+
+    try {
+        const metadataFilePath = path.join(actionPackagePath.fsPath, "metadata.json");
+
+        if (!fs.existsSync(metadataFilePath)) {
+            await createMetadata(actionPackagePath);
+        }
```
[the remark this answers is not in the document]
Comment (mmokko) [reply sema4ai/vscode-client/src/robo/actionPackage.ts:784]: Yeah that's a better idea 👍 I'll implement it.

File: sema4ai/package.json [hunk 2]
```diff
@@ -888,12 +889,17 @@
             },
             {
                 "command": "sema4ai.actionServerPackageMetadata",
-                "title": "Create Action Package metadata file",
+                "title": "Create Action Package metadata (metadata.json)",
```
Comment (custompointofview) [inline sema4ai/package.json:892]: Probably package.json needs to be regenerated

**Review by custompointofview — APPROVED** [review]
LGTM

**Pull request merged**